### PR TITLE
Feat/next

### PR DESCRIPTION
**Pull request by hirsch88**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: projects/lib/src/form/form-wizard/wizard-overview/wizard-overview.component.scss [hunk 1]
```diff
@@ -1 +1 @@
-@import 'node_modules/@baloise/design-system-components/src/styles/global.utilities.scss';
+@import '~@baloise/design-system-components/src/styles/global.utilities';
```
Comment (nobilo) [inline projects/lib/src/form/form-wizard/wizard-overview/wizard-overview.component.scss:1]: new angular complains for somthing like this:
```
Warning: 'C:\Users\Public\dev\repos\mybaloise-business\dashboard-enterprise-client\node_modules\@baloise\design-system-next-components\src\styles\global.helpers.sass' imports '~@baloise/design-system-next-css/scss/baloise-design-sys
tem.helpers.sass' with a tilde. Usage of '~' in imports is deprecated.

````
maybe we should leave import with node_modules?

File: projects/lib/package.json [hunk 2]
```diff
@@ -5,11 +5,23 @@
     "@angular/common": ">=12.0.0",
     "@angular/core": ">=12.0.0",
     "@angular/forms": ">=12.0.0",
```
Comment (nobilo) [inline projects/lib/package.json:7]: should this also be raised on 12, or maybe 13?

**Review by nobilo — CHANGES_REQUESTED** [review]

File: projects/lib/package.json (hunk 2 above)
Comment (hirsch88) [reply projects/lib/package.json:7]: I do not know. You are the ngProfis 😘

Comment (nobilo) [reply projects/lib/package.json:7]: will do it :)

**Review by nobilo — APPROVED** [review]

**Review by lazaki — APPROVED** [review]

**Pull request merged**